robottelo/manifests.py

```diff
@@ -32,7 +32,10 @@ def _download_manifest_info(self, name='default'):
         """Download and cache the manifest information."""
         if self.template is None:
             self.template = {}
-        self.template[name] = requests.get(settings.fake_manifest.url[name]).content
+        if name == "golden_ticket":
+            self.template[name] = requests.get(settings.fake_manifest.golden_ticket).content
```

Comment:
I'm not an expert here, but that seems to call for solution similar to one that we have for `sattools_repo` or `rhel8_os`, where you can pass multiple named arguments to single key.
What do you think?

Reply:
exactly what @mirekdlugosz said.
The multi-manifest support is already there, just define it in robottelo.properties like so:
https://github.com/SatelliteQE/robottelo/pull/6724

Reply:
I initially tried that but ran into some issues when trying to call it by name or url. I either got an error or it just read in the name and not the content. Any ideas on why I couldn't use multi-support for manifests?

Reply:
@ColeHiggins2 below seems to work for me:
in robottelo.properties:
```
[fake_manifest]
url=default=http://example.com/manifests/manifest-latest.zip,golden_ticket=http://example.com/manifests/manifest-golden-ticket.zip
```
In Python shell:
```python
>>> from robottelo.config import settings
>>> settings.configure()
>>> settings.fake_manifest.url
{'default': 'http://example.com/manifests/manifest-latest.zip', 'golden_ticket': 'http://example.com/manifests/manifest-golden-ticket.zip'}
>>> settings.fake_manifest.url['golden_ticket']
'http://example.com/manifests/manifest-golden-ticket.zip'
```

Looking at code, most of (or all of) functions in `robottelo/manifests` can take `name` argument that should point to your name. Like:
```python
with manifests.clone(name="golden_ticket") as manifest:
    # do something with manifest object
```

Reply:
I was able to recreate this. It seems that I initially modified the properties file incorrectly. Thanks for the help.